Sources/WalletConnectPush/Client/Wallet/WalletPushClient.swift

```diff
@@ -84,10 +83,12 @@ public class WalletPushClient {
         self.notifyUpdateRequester = notifyUpdateRequester
         self.notifyUpdateResponseSubscriber = notifyUpdateResponseSubscriber
         self.notifyProposeResponder = notifyProposeResponder
+        self.subscriptionsStore = subscriptionsStore
         setupSubscriptions()
     }
 
     public func subscribe(metadata: AppMetadata, account: Account, onSign: @escaping SigningCallback) async throws {
+        try await subscriptionsStore.initialize(for: account)
```

Comment:
so from here I understand that one store can initialise multiple sync services, correct?

Reply:
yes, one store can work with multiple accounts. On `initialize` we're doing sync topic derivation and subscribing for sync topic